packages/perseus/src/util/graphie.ts

```diff
@@ -1675,10 +1675,34 @@ const setLabelMargins = function (span: HTMLElement, size: Coord): void {
             marginTop: -height / 2 - y * scale,
         });
     } else {
+        const $container = $span.closest(".svg-image");
+        // Ensuring that the line-height of the text doesn't throw off placement of the text element.
+        // Inherited line-height values can really mess up placement.
```

Comment:
How does setting it to `normal` help? (for a CSS newb)

Reply:
Most default `line-height` settings are 1.2. However, that value isn't appropriate for all fonts. So, setting this property to `"normal"` sets it to the preferred value for the specified `font-family`.